internal/flag/flags.go

```diff
@@ -309,6 +309,16 @@ const (
 	FrequencyInterval                         = "frequencyInterval"                         // FrequencyInterval flag
 	RetentionUnit                             = "retentionUnit"                             // RetentionUnit flag
 	RetentionValue                            = "retentionValue"                            // RetentionValue flag
-	OIDCType                                  = "oidcType"                                  // OidcType flag
 	FederationSettingsID                      = "federationSettingsId"                      // FederationSettingsId flag
+	OIDCType                                  = "oidcType"                                  // OidcType flag
+	IdpType                                   = "idpType"                                   // IdpType flag
+	Audience                                  = "audience"                                  // Audience flag
+	AuthorizationType                         = "authorizationType"                         // AuthorizationType flag
+	ClientID                                  = "clientId"                                  // ClientId flag
+	DisplayName                               = "displayName"                               // DisplayName flag
+	GroupsClaim                               = "groupsClaim"                               // GroupsClaim flag
+	UserClaim                                 = "userClaim"                                 // UserClaim flag
+	IssuerURI                                 = "issuerUri"                                 // IssuerURI flag
+	AssociatedDomains                         = "associatedDomains"                         // AssociatedDomains flag
+	RequestedScopes                           = "requestedScopes"                           // RequestedScopes flag
```

Comment:
```suggestion
	AssociatedDomains                         = "associatedDomain"                         // AssociatedDomains flag
	RequestedScopes                           = "requestedScope"                           // RequestedScopes flag
```
singular even if repeated